.github/workflows/c-cpp.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         key: ccache-macos-build-${{ github.sha }}
         restore-keys: ccache-macos-build-
     - name: install dependencies
-      run: HOMEBREW_NO_AUTO_UPDATE=1 brew bundle --file=contrib/apple/brew
+      run: HOMEBREW_NO_AUTOUPDATE=1 brew install autoconf ccache autogen automake binutils coreutils libsodium openssl pkg-config boost hidapi ldns expat libunwind-headers readline
```

Comment:
why did you specify static libs to install instead of updating ```brew``` file?
such change require any libraries update changes to be made at more places to do not confuse anybody which is current

Reply:
Yes this can be done like that